src/AppKit/Enums.cs

```diff
@@ -3009,4 +3013,48 @@ public enum NSWorkspaceAuthorizationType : long  {
 		SetAttributes,
 		ReplaceFile,
 	}
+
+	[Mac (10,16)]
+	[Native]
+	public enum NSTableViewStyle : long
+	{
+		Automatic,
+		FullWidth,
+		Inset,
+		SourceList
+	}
+
+	[Mac (10,16)]
+	[Native]
+	public enum NSTitlebarSeparatorStyle : long
+	{
+		Automatic,
+		None,
+		Line,
+		Shadow
+	}
+
+	[Mac (10,16)]
+	[Native]
+	public enum NSWindowToolbarStyle : long
+	{
+		Automatic,
+		Expanded,
+		Preference,
+		Unified,
+		UnifiedCompact
```

Comment:
minor: add a `,` so any future addition won't make the last value part of the diff